internal/extramanifest/extramanifest.go

```diff
@@ -179,79 +192,149 @@ func ApplyExtraManifest(ctx context.Context, dc dynamic.Interface, restMapper me
 	return nil
 }
 
-func (h *EMHandler) ValidateExtraManifestConfigmaps(ctx context.Context, content []ibuv1.ConfigMapRef, ibu *ibuv1.ImageBasedUpgrade) error {
+// ValidateExtraManifestConfigmaps validates IBU extramanifest configmaps and returns validation warning and error.
+// It iterates over all manifests included in the configmaps and perform Kubernetes Dry-run.
+// Errors from Dry-run apply are considered warnings. This also includes cases where CRDs are missing from the current
```

Comment:
I think should say something like "Most errors from Dry-run apply..." ?

Reply:
yeah, I think I should make it more accurate